CMakeLists.txt

```diff
@@ -53,6 +53,12 @@ install(DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/ DESTINATION ${CMAKE_INSTALL
 install(DIRECTORY pd-help/ DESTINATION ${CMAKE_INSTALL_PREFIX})
 
 #Dependencies: availability
+#Non-standard openssl path on OSX
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
```

Comment:
There's also [FindOpenSSL](https://cmake.org/cmake/help/v3.0/module/FindOpenSSL.html) and [pkg_search_module](http://stackoverflow.com/a/26534464). At the very least, I would try "/usr/local/opt/openssl" before asking to pass a constant.